ui-components/src/locales/ar.json

```diff
@@ -1331,9 +1331,17 @@
         "title": "تقدم بطلب للحصول على سكن ميسور التكلفة في",
         "seeMoreOpportunities": "شاهد المزيد من فرص الإسكان الإيجاري والملكية",
         "viewAdditionalHousing": "عرض فرص وموارد الإسكان الإضافية",
+        "bedrooms": {
+            "studios": "استوديو متاح(%{smart_count}) |||| استوديو متاح(%{smart_count})",
+            "numBed": "(%{smart_count}) %{num_bed} غرفة نوم المتاحة |||| (%{smart_count}) %{num_bed} غرف نوم المتاحة",
+            "fourPlusBed": "(%{smart_count}) 4+ غرفة نوم المتاحة |||| (%{smart_count}) 4+ غرف نوم المتاحة"
```

Comment:
Do you need to reverse where the numbers are here?

Reply:
I couldn't get the numbers to move (RTL mixing with LTR is pretty tricky), but given we're going to have to change this string anyway based on the new mocks, I'm going to submit for now and see if I can fix it with the new strings.